scripts/bootstrapOrUpdate.ts

```diff
@@ -17,6 +17,7 @@ export async function bootstrapOrUpdate(path: string) {
     await updateDatabase(data)
     process.exit(0)
   } catch (e) {
+    console.log(e.message)
```

Comment:
Is this a left behind `console.log`?

Reply:
nah that's there on purpose, this is run via various CLI tasks and I wanted to leave a trace of what went wrong if it fails. Logging that output out feels appropriate in this context?

Reply:
Makes sense.